app/src/main/res/layout/fragment_home.xml

```diff
@@ -1,121 +1,115 @@
-                android:text="@string/share_the_app"
-                android:textColor="@color/white" />
+        <Button
+            android:id="@+id/notify_others_button"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="@dimen/padding_normal"
+            android:text="@string/notify_others_button"
+            android:textSize="16sp" />
 
-            <TextView
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:paddingHorizontal="32dp"
-                android:text="@string/main_subtext_1"
-                android:textAlignment="center" />
+        <TextView
+            android:id="@+id/notify_others_button_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="@dimen/padding_normal"
+            android:text="@string/notify_others_button_text"
+            android:textAlignment="center" />
 
-            <Button
-                android:id="@+id/tested_button"
-                style="?attr/materialButtonOutlinedStyle"
-                android:layout_width="match_parent"
-                android:layout_height="56dp"
-                android:layout_marginHorizontal="32dp"
-                android:layout_marginTop="16dp"
-                android:text="@string/text_for_covid_text" />
+        <Button
+            android:id="@+id/share_app_button"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="@dimen/padding_normal"
+            android:layout_marginTop="@dimen/padding_normal"
+            android:text="@string/share_app_button" />
 
-            <TextView
-                android:id="@+id/tested_button_text"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:paddingHorizontal="32dp"
-                android:text="@string/main_subtext_2"
-                android:textAlignment="center" />
+        <TextView
+            android:id="@+id/tester_id"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="16dp"
+            android:paddingHorizontal="32dp"
+            android:textAlignment="center"
+            android:visibility="gone" />
 
-            <TextView
-                android:id="@+id/tester_id"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:paddingHorizontal="32dp"
-                android:layout_marginTop="16dp"
-                android:text=""
-                android:visibility = "gone"
-                android:textAlignment="center" />
-        </LinearLayout>
-    </ScrollView>
+        <View
```

Comment:
what's that for?

Reply:
this dump view pushes powered by content to the bottom of screen if device has enough space